diff -ur libressl-2.5.5.orig/apps/openssl/Makefile.am libressl-2.5.5/apps/openssl/Makefile.am
--- libressl-2.5.5.orig/apps/openssl/Makefile.am	2017-07-05 20:18:20.000000000 -0700
+++ libressl-2.5.5/apps/openssl/Makefile.am	2017-09-25 02:18:40.882395079 -0700
@@ -2,7 +2,7 @@
 
 bin_PROGRAMS = openssl
 
-dist_man_MANS = openssl.1
+dist_man_MANS = libressl.1
 
 openssl_LDADD = $(abs_top_builddir)/ssl/libssl.la
 openssl_LDADD += $(abs_top_builddir)/crypto/libcrypto.la
diff -ur libressl-2.5.5.orig/apps/openssl/Makefile.in libressl-2.5.5/apps/openssl/Makefile.in
--- libressl-2.5.5.orig/apps/openssl/Makefile.in	2017-07-09 04:00:06.000000000 -0700
+++ libressl-2.5.5/apps/openssl/Makefile.in	2017-09-25 02:17:56.899233315 -0700
@@ -379,7 +379,7 @@
 AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/include/compat \
 	-DLIBRESSL_INTERNAL -D__BEGIN_HIDDEN_DECLS= \
 	-D__END_HIDDEN_DECLS=
-dist_man_MANS = openssl.1
+dist_man_MANS = libressl.1
 openssl_LDADD = $(abs_top_builddir)/ssl/libssl.la \
 	$(abs_top_builddir)/crypto/libcrypto.la $(PLATFORM_LDADD) \
 	$(PROG_LDADD)
